cmd/cmd.go

```diff
@@ -99,6 +100,7 @@ func initConfig() {
 			fmt.Fprintln(os.Stderr, err)
 		}
 	}
+	fmt.Println("Using config file:", viper.ConfigFileUsed())
```

Comment:
Overall, the code patch looks fine. There are no obvious bug risks or security vulnerabilities.

One improvement suggestion would be to remove the print statements used for debugging purposes (e.g. "home dir", "Using config file"). These print statements can be useful during development, but they should not be left in production code as they can leak sensitive information and clutter log files.

Another improvement suggestion would be to add more error handling for cases where the config file cannot be found or read. Currently, the code just prints the error message to stderr, but it would be better to handle the error gracefully and provide more detailed feedback to the user.